internal/gotest/gotest.go

```diff
@@ -102,10 +108,8 @@ func Run(ctx context.Context, dir string, args []string, verbose bool, parallel
 		case actionSkip:
 			result.Status = config.Skip
 		case actionBench, actionCont, actionOutput, actionPause, actionRun:
-			fallthrough
-		default:
-			result.Status = config.Unknown
```

Comment:
I just realized that his file was completely correct: the test run must end with one of pass/skip/fail. If any other event is received, the test is in an unknown state.

Let's revert those changes